kvbc/src/categorization/kv_blockchain.cpp

```diff
@@ -13,12 +13,17 @@
 
 #include "categorization/kv_blockchain.h"
 #include "bcstatetransfer/SimpleBCStateTransfer.hpp"
+#include "bftengine/ControlStateManager.hpp"
+#include "json_output.hpp"
 
 #include <stdexcept>
 
+#define getName(var) #var
```

Comment:
I prefer not to litter every cpp file with this macro. Also, `getName(x)` is longer than just `"x"`. Why even use this macro at all?

Reply:
Yes, I agree, I added as the same was being used in `concord-ctl` related functionality in multiple places already. So, my understanding was that, it was being used for code readability and maybe to handle future changes in the names of variables. But sure, I changed the usage to direct name of the variable, its more precise and no need of the macro.